src/toolkit/facility_cost.cc

```diff
@@ -0,0 +1,25 @@
+#include "facility_cost.h"
+
+#include <math.h>
+#include <stdio.h>
+#include <iomanip>
+#include <sstream>
+
+namespace cyclus {
+namespace toolkit {
+
+// Interested in thoughts on the member initializer list
+FacilityCost::FacilityCost() : capital_cost_(0) {}
+
+FacilityCost::FacilityCost(double capital_cost) {
+  capital_cost_ = capital_cost;
+}
+
+FacilityCost::~FacilityCost() {}
+
+void FacilityCost::EnumerateCosts(){
+    std::cout<<capital_cost_<<std::endl;
```

Comment:
Probably best for this to return a string rather than writing to stdout

Reply:
You make a good point. It may be best to just write a getter for this value and call that the test function. EnumerateCosts was just a stand-in that showed you could give it a function, but I don't know that it really serves a purpose in the grand scheme of what we'd want to do with this.

Reply:
It is probably useful to have a way to dump all the data (that will eventually be there) in a nice string format.  Some times this is just a `str()` function that is called to convert to a string.  (recall this from Tricycle)

Reply:
Sounds good. I made EnumerateCosts return a string.